### PR TITLE
ParadeDB Enterprise

### PR DESCRIPTION
**Pull request by itay-grudev**

### CODE REVIEW

File: .github/workflows/tests-cluster-chainsaw.yaml [hunk 1]
```diff
@@ -75,3 +75,7 @@ jobs:
 
       - name: Run Kyverno/Chainsaw
         run: chainsaw test
+        env:
+          PARADEDB_ENTERPRISE_DOCKER_USERNAME: ${{ secrets.PARADEDB_ENTERPRISE_DOCKER_USERNAME }}
+          PARADEDB_ENTERPRISE_DOCKER_PAT: ${{ secrets.PARADEDB_ENTERPRISE_DOCKER_PAT }}
```
Comment (philippemnoel) [inline .github/workflows/tests-cluster-chainsaw.yaml:80]: We actually already have some of these in CI we use in other places, I'll swap them for them @itay-grudev

Comment (philippemnoel) [reply .github/workflows/tests-cluster-chainsaw.yaml]: Done

File: charts/paradedb/test/paradedb-enterprise/01-paradedb-NCC-1701-D_cluster-assert.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: paradedb-ncc-1701-d
```
Comment (philippemnoel) [inline charts/paradedb/test/paradedb-enterprise/01-paradedb-NCC-1701-D_cluster-assert.yaml:4]: @itay-grudev Everything looks good to me aside from a question: What does this `NCC-1701` naming means?

**Comment by philippemnoel** [comment]
The failing check is because we haven't published 0.12.1 enterprise yet -- It seems the enterprise build isn't hardcoded to a specific version like the other tests?

File: charts/paradedb/test/paradedb-enterprise/01-paradedb-NCC-1701-D_cluster-assert.yaml (hunk 2 above)
Comment (itay-grudev) [reply charts/paradedb/test/paradedb-enterprise/01-paradedb-NCC-1701-D_cluster-assert.yaml:4]: This is the test for the Enterprise version...

image

**Pull request merged**